src/item/GoatHorn.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+use pocketmine\data\runtime\RuntimeDataWriter;
+use pocketmine\math\Vector3;
+use pocketmine\player\Player;
+
+class GoatHorn extends Item implements Releasable{
+
+	private GoatHornType $goatHornType;
+
+	public function __construct(ItemIdentifier $identifier, string $name){
+		$this->goatHornType = GoatHornType::PONDER();
+		parent::__construct($identifier, $name);
+	}
+
+	protected function encodeType(RuntimeDataWriter $w) : void{
+		$w->goatHornType($this->goatHornType);
+	}
+
+	public function getType() : GoatHornType{ return $this->goatHornType; }
```

Comment:
tbh I prefer, that the code is consistent, if it is common (as observed in several places in the PM code) I would expect by intuition that to access the variants you use `getType`, so for consistency it would be strange to change the format.

Reply:
My intent is to replace the getType() in other places with more specific APIs in the future. It currently only appears in 2-3 places, and the more specific style is much more common.